### PR TITLE
French translation update

### PR DESCRIPTION
**Pull request by atomic83GitHub**
I update the French translation.
Can be reviewed.

### CODE REVIEW

File: locales/fr_FR/pcsx2_Iconized.po [hunk 1]
```diff
@@ -44,7 +41,8 @@ msgstr ""
 msgid ""
 "Failed to read/write NMV/MEC file. Check your bios setup/permission settings"
 msgstr ""
-
+"Echec de l'écriture/lecture du fichier NMV/MEC "
+Vérifiez vos paramètres/permissions dans votre bios."
```
Comment (lightningterror) [inline locales/fr_FR/pcsx2_Iconized.po]: Quotation mark.

File: locales/fr_FR/pcsx2_Main.po [hunk 2]
```diff
@@ -1720,16 +1718,16 @@ msgid ""
...
 
 #: pcsx2/gui/MainFrame.cpp:785
-msgid "Window Settings..."
-msgstr "Paramètres du mode fenêtré..."
+msgid "&Window Settings..."
+msgstr "Paramètres du mode &fenêtré..."
 
 #: pcsx2/gui/MainFrame.cpp:786
 msgid "Modify window and appearance options, including aspect ratio."
 msgstr "Modifie la fenêtre et l'apparence dont le ratio d'aspect (4:3, 16:9)."
```
Comment (FlatOutPS2) [inline locales/fr_FR/pcsx2_Main.po]: You could remove "(4:3, 16:9)" here. It's not in the original message and doesn't include all options(stretch, which could be any aspect ratio).

File: locales/fr_FR/pcsx2_Main.po [hunk 3]
```diff
@@ -28,6 +25,7 @@ msgstr "Erreur critique."
 
 #: common/include/Utilities/Exceptions.h:255
 msgid "Parse error"
+msgstr "Erreur d'annalyse"
 msgstr "erreur d'analyse"
```
Comment (FlatOutPS2) [inline locales/fr_FR/pcsx2_Main.po]: You forgot to remove the old line. ;)

**Comment by lightningterror** [comment]
You don't need to update your branches every time a new commit is merged in to master. Let it be, github is smart :wink: 
The only time you wanna update the code is if a new commit created a merge conflict on your PR.
Anyway can you do a squash rebase ?

**Comment by gregory38** [comment]
We can squash commit during the "merge" (from the web interface) but it isn't possible here due to conflict

File: locales/fr_FR/pcsx2_Iconized.po [hunk 4]
```diff
@@ -668,7 +666,7 @@ msgid ""
 "-1 - Reduces the EE's cyclerate to about 75%.  Mild speedup for most games "
 "with high compatibility."
 msgstr ""
-"-1 - Diminue la fréquence du processeur EE d'environ 33% : amélioration des "
+"-1 - Diminue la fréquence du processeur EE d'environ 33%. Amélioration des "
```
Comment (FlatOutPS2) [inline locales/fr_FR/pcsx2_Iconized.po]: Don't you think it's weird that the French translation reduces the cyclerate by 33% while the English original only reduces it by 25%... 😛

File: locales/fr_FR/pcsx2_Main.po [hunk 5]
```diff
@@ -28,7 +25,7 @@ msgstr "Erreur critique."
 
 #: common/include/Utilities/Exceptions.h:255
 msgid "Parse error"
-msgstr "erreur d'analyse"
+msgstr "Erreur d'annalyse"
```
Comment (FlatOutPS2) [inline locales/fr_FR/pcsx2_Main.po]: `analyse` was correct?

File: locales/fr_FR/pcsx2_Iconized.po [hunk 6]
```diff
@@ -44,7 +41,10 @@ msgstr ""
 msgid ""
 "Failed to read/write NMV/MEC file. Check your bios setup/permission settings"
 msgstr ""
-"Échec de la lecture/écriture du fichier NMV/MEC. Verifiez vos paramètres/permissions dans votre bios "
+
+"Echec de l'écriture/lecture du fichier NMV/MEC "
+Vérifiez vos paramètres/permissions dans votre bios."
```
Comment (FlatOutPS2) [inline locales/fr_FR/pcsx2_Iconized.po]: You still need to add a " here to fix the build.

**Comment by lightningterror** [comment]
@FlatOutPS2 This one is ready no ?

**Comment by FlatOutPS2** [comment]
Other than the weird merge master branch commit that contains the fixes, it seems fine.

**Pull request merged**